### ISSUE TITLE
Can ESP32 change Station and AP Power?

### ISSUE THREAD

**Issue by uecken** [espressif/esp32-wifi-lib#27]
Hi, I want to control Station and AP Power.
I assumes that ESP32-SoftAP communicates with ESP32-Station and other WiFi device.

The following function seems to change AP Power.
Can this function also change station power? 
[esp_wifi_set_max_tx_power](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/network/esp_wifi.html#_CPPv425esp_wifi_set_max_tx_power6int8_t)

In addition, can STA/AP change power real time?

**Comment by jack0c** [comment]
Yes, this API changes global Wi-Fi TX power, and can change the power real time.
